hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerPrepare.java

```diff
@@ -175,7 +184,11 @@ public void testPrepareDownedOM() throws Exception {
 
   @Test
   public void testPrepareWithRestart() throws Exception {
+    // Create fresh cluster for this test to prevent timeout from restarting
```

Comment:
@JyotinderSingh  Do we need a new cluster? I ran 10 times in my laptop without a new cluster(without init() and setup() calls ) and passed consistently. Please double check it. Also, if we need to setup new cluster then please shutdown the previously instantiated cluster by invoking `shutdown();` function before `init()`.
```
    // Create fresh cluster for this test to prevent timeout from restarting
    // modified cluster.
    init();
    setup();
```

Reply:
This was working without the need for a new cluster on my local system as well. But yesterday this test timed out on CI (while executing < 10s on my local) - and I thought that this might solve the issue.
Might have been some CI related issue. Will remove this and run checks again to see if it still persists.

Reply:
OK, good catch. I'm OK to re-setup cluster after`#shutdown()` first one. 
BTW, can you point me to the build link, we can download the build artifact and check the logs to understand the behavior.

Reply:
Here's the link to the CI run: https://github.com/apache/ozone/runs/3734530549#step:4:2979